collectors/cluster_usage.go

```diff
@@ -106,29 +106,32 @@ func (c *ClusterUsageCollector) collect() error {
 		return err
 	}
 
-	tot, err := stats.Stats.TotalBytes.Float64()
+	var totBytes, usedBytes, availBytes, totObjects float64
+
+	totBytes, err = stats.Stats.TotalBytes.Float64()
 	if err != nil {
-		return err
+		log.Println("[ERROR] cannot extract total bytes:", err)
```

Comment:
I have changed the logic to show blips to separate stats-related failures with network-related ones. This case should almost never be triggered and there should be a strong visible sign on our graph if it ever does.

Easy enough to add prefixes for now but good to have that option once we start using them extensively.